python/python/glide/async_commands/server_modules/json.py

```diff
@@ -204,6 +204,56 @@ async def get(
     return cast(TJsonResponse[Optional[bytes]], await client.custom_command(args))
 
 
+async def mget(
+    client: TGlideClient,
+    keys: List[TEncodable],
+    path: Optional[TEncodable] = None,
```

Comment:
`path` is required, not optional, according to all documentations we have access to. Please correct this, including all corresponding docs, and remove the test case without `path`.

Reply:
You're right. Fixed it now.